README.es.md

```diff
@@ -0,0 +1,167 @@
+<p align="center" style="text-align: center">
+  <img src="./assets/images/logo-universal.png" width="55%"><br/>
+</p>
+
+<p align="center">
+  Construye aplicaciones de escritorio usando Go y tecnologías web.
+  <br/>
+  <br/>
+  <a href="https://github.com/wailsapp/wails/blob/master/LICENSE">
+    <img alt="GitHub" src="https://img.shields.io/github/license/wailsapp/wails"/>
+  </a>
+  <a href="https://goreportcard.com/report/github.com/wailsapp/wails">
+    <img src="https://goreportcard.com/badge/github.com/wailsapp/wails" />
+  </a>
+  <a href="https://pkg.go.dev/github.com/wailsapp/wails">
+    <img src="https://pkg.go.dev/badge/github.com/wailsapp/wails.svg" alt="Go Reference"/>
+  </a>
+  <a href="https://github.com/wailsapp/wails/issues">
+    <img src="https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat" alt="CodeFactor" />
+  </a>
+  <a href="https://app.fossa.com/projects/git%2Bgithub.com%2Fwailsapp%2Fwails?ref=badge_shield" alt="FOSSA Status">
+    <img src="https://app.fossa.com/api/projects/git%2Bgithub.com%2Fwailsapp%2Fwails.svg?type=shield" />
+  </a>
+  <a href="https://github.com/avelino/awesome-go" rel="nofollow">
+    <img src="https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg" alt="Awesome" />
+  </a>
+  <a href="https://discord.gg/BrRSWTaxVK">
+    <img alt="Discord" src="https://dcbadge.vercel.app/api/server/BrRSWTaxVK?style=flat"/>
+  </a>
+  <br/>
+  <a href="https://github.com/wailsapp/wails/actions/workflows/build-and-test.yml" rel="nofollow">
+    <img src="https://img.shields.io/github/actions/workflow/status/wailsapp/wails/build-and-test.yml?branch=master&logo=Github" alt="Build" />
+  </a>
+  <a href="https://github.com/wailsapp/wails/tags" rel="nofollow">
+    <img alt="GitHub tag (latest SemVer pre-release)" src="https://img.shields.io/github/v/tag/wailsapp/wails?include_prereleases&label=version"/>
+  </a>
+</p>
+
+<div align="center">
+<strong>
+<samp>
+
+[English](README.md) · [简体中文](README.zh-Hans.md) · [日本語](README.ja.md) ·
+[한국어](README.ko.md) · [Spanish](README.es.md)
```

Comment:
Would it be better to use "Español" here?

Reply:
Yep, this makes sense, thanks for the feedback! Also added the Español link to the other READMEs as well